integration/load_balancer/run.sh

```diff
@@ -10,7 +10,7 @@ export PGHOST=127.0.0.1
 export PGDATABASE=postgres
 export PGPASSWORD=postgres
 
-docker-compose up -d
+docker compose up -d
```

Comment:
`docker-compose` is deprecated.
`docker compose` has replaced it.

`docker-compose` does not work on my recent version of `docker` desktop.
`docker compose` works on the version of docker in our CI.

win-win?
I suppose it will work for a every engineer's machine too. Can rollback this change if that is not the case